### PR TITLE
intial svelte support

### PR DESCRIPTION
**Pull request by mosheduminer**

### CODE REVIEW

File: packages/svelte/src/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+)};
+          </script>
+      `;
+
+    next();
+  } catch (error) {
+    res.locals.error = error;
+
+    next('route');
+  }
```
Comment (maniator) [inline packages/svelte/src/index.js:28]: we should probably make some shared helper for this repeat patter @ some point (maybe a ssr-aas-utils package?)

I can put on the kanban

Comment (mosheduminer) [reply packages/svelte/src/index.js:28]: I agree. This code also lowered the test coverage...

Comment (maniator) [reply packages/svelte/src/index.js:28]: ha -- we can ignore coverage report for now -- If you want, maybe write tests for everything in this PR? (if possible)

File: packages/svelte/jest.config.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const base = require('../../jest.config.base.js');
+
+module.exports = {
+  ...base,
+  name: '@serveside/ssr-aas-core',
```
Comment (maniator) [inline packages/svelte/jest.config.js:5]: name is wrong here

File: packages/svelte/src/index.js (hunk 1 above)
Comment (mosheduminer) [reply packages/svelte/src/index.js:28]: Yeah, I'll add some tests.

File: packages/svelte/package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@serveside/ssr-aas-svelte",
+  "version": "0.0.1-alpha.4",
+  "description": "> TODO: description",
+  "author": "Moshe Uminer <47164590+mosheduminer@users.noreply.github.com>",
+  "homepage": "https://github.com/serveside/ssr-aas#readme",
+  "license": "MIT",
+  "main": "dist/core.js",
```
Comment (maniator) [inline packages/svelte/package.json:8]: I just fixed this in main for the other packages, make sure to update

```suggestion
  "main": "dist/index.js",
  "publishConfig": {
    "access": "public"
  },
```

**Pull request merged**